teps/0091-trusted-resources.md

```diff
@@ -377,13 +386,44 @@ metadata:
   name: feature-flag
   namespace: tekton-pipelines
 data:
-  verification-policy: enforce
+  enable-trusted-resource: true
 ```
 
-`verification-policy`. (Optional, `enforce`, `warn` or `skip`, default `skip`):
- * `enforce`: Failing verification will mark the taskruns/pipelineruns as failed.
- * `warn`: Log warning but don't fail the taskruns/pipelineruns.
- * `skip`: Directly skip the verification.
+`enable-trusted-resource`. (Optional, `true` or `false`, default `false`):
+ * `true`: Failing verification will mark the taskruns/pipelineruns as failed.
+ * `false`: Directly skip the verification.
+
+
+ ```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-trusted-resources
+  namespace: tekton-pipelines
+data:
+  no-match-policy: "deny"
+```
+
+`no-match-policy`. (Optional, `allow`, `deny` or `warn`, default `deny`):
```

Comment:
You probably want to default to allow, at least while still in alpha - this is the other piece that will let you drop the 
feature flag.

This may end up being a feature we always start at allow/warn - we probably don't want to flip on enforcement until a user is ready.